frontend/src/app/order/order/order.component.html

```diff
@@ -0,0 +1,167 @@
+<app-nav-spacer></app-nav-spacer>
+<app-header
+  heading="Tarif bestellen"
+  [breadcrumbs]="breadcrumbs"
+  [whiteBg]="true">
+</app-header>
+<app-container>
+  <div *ngIf="!loading">
+    <h3 class="text-2xl text-gray-600">Gewählter Tarif</h3>
+    <p class="text-gray-600 py-4">Bitte überprüfen Sie, ob Sie den richtigen Tarif gewählt haben.</p>
+    <app-plan-detail [plan]="plan" [estimatedUsage]="estimatedUsage"></app-plan-detail>
+    <ng-container *ngIf="currentStep === 1">
+      <div class="bg-white shadow overflow-hidden sm:rounded-lg mb-10 p-4 md:flex items-center">
+        <app-zero-data
+          class="flex-1"
+          image="wind"
+          description="Geben Sie hier Ihren geschätzen jährlichen Verbrauch an. Alternativ können Sie auch die Personenzahl angeben."
+          heading="Geschätzter Verbrauch"></app-zero-data>
+        <div class="flex-1">
+            <app-usage-people-counter [currentEstimatedUsage]="estimatedUsage"
+                                      (estimatedUsage)="estimatedUsage = $event"></app-usage-people-counter>
```

Comment:
An dieser Stelle wird in deinem Video der geschätzte Verbrauch, aber nicht die eingegebene Personenzahl angezeigt.

Reply:
![Bildschirmfoto 2020-12-21 um 16 58 48](https://user-images.githubusercontent.com/11008743/102795956-ce088480-43ad-11eb-911a-68efcbf2c1e1.png)
Die Component `app-usage-people-counter` rendert sowohl die Personen- als auch Verbrauchsanzahl